AATAMS/AATAMS_sattag_dm/lftp_sync.sh

```diff
@@ -23,10 +23,15 @@ sync_files() {
 # main
 main() {
     mkdir -p $ZIPPED_DIR
-    local tmp_lftp_output_file=`mktemp`
-    sync_files $tmp_lftp_output_file
 
-    mv $tmp_lftp_output_file $INCOMING_DIR/AATAMS/AATAMS_SATTAG_DM/aatams_sattag_dm_lftp.`date +%Y%m%d-%H%M%S`.log
+    # look for empty dir
+    if [ "$(ls -A $ERROR_DIR/AATAMS_SATTAG_DM)" ]; then
+         echo "Unable to run lftp as $ERROR_DIR/AATAMS_SATTAG_DM is not empty"
```

Comment:
Currently, the lftp/rsync runs as set up in the cron (similar thing for both AATAMS SATTAG DM and ARGO). It adds/removes files to/from $WIP_DIR. Every 'additions' and 'deletion' is written in a manifest file which gets pushed to the incoming directory.

In the case the manifest file doesn't get processed (this was the case with ARGO), more manifest files are still generated. Each manifest file will be unique, containing different additions and deletions. In the case these manifest files landed in the ERROR_DIR, it is impossible to run them again, because some files have disappeared physically from the $WIP_DIR ($WIP_DIR is a FS and the increment is done via the manifest).

It's exactly what happens with ARGO. I have to look now at the manifest files kind of manually to generate semi-manually a new manifest file combining the last 2....

So this PR prevents the cron to run in case a file has landed to ERROR_DIR.

makes sense ?